source/iterators.tex

```diff
@@ -7391,7 +7411,8 @@
 
 \indexlibrary{\idxcode{crbegin(const C\& c)}}%
 \begin{itemdecl}
-template<class C> constexpr auto crbegin(const C& c) -> decltype(std::rbegin(c));
+template<class C> constexpr auto crbegin(const C& c) noexcept(noexcept(c.crbegin()))
+  -> decltype(std::rbegin(c));
```

Comment:
```suggestion
template<class C> constexpr auto crbegin(const C& c) noexcept(noexcept(std::rbegin(c)))
  -> decltype(std::rbegin(c));
```
Same mistake of using the member functions instead of the `std::` variant in a few other places.